enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/core/consensus/outcome/Outcome.java

```diff
@@ -226,7 +230,7 @@ public String toString()
                ", followerStates=" + followerStates +
                ", shipCommands=" + shipCommands +
                ", electedLeader=" + electedLeader +
-               ", steppingDown=" + steppingDown +
+               ", steppingDown=" + steppingDownInTerm.map( t -> "true, steppingDownInTerm=" + t ).orElse( "false" ) +
```

Comment:
You might want to not over-complicate it and keep the toString of the Optional perhaps?